homeassistant/components/airnow/sensor.py

```diff
@@ -85,6 +89,16 @@ class AirNowEntityDescription(SensorEntityDescription, AirNowEntityDescriptionMi
         value_fn=lambda data: data.get(ATTR_API_O3),
         extra_state_attributes_fn=None,
     ),
+    AirNowEntityDescription(
+        key=ATTR_API_STATION,
+        translation_key="station",
+        icon="mdi:blur",
+        value_fn=lambda data: data.get(ATTR_API_STATION),
+        extra_state_attributes_fn=lambda data: {
+            "latitude": data[ATTR_API_STATION_LATITUDE],
+            "longitude": data[ATTR_API_STATION_LONGITUDE],
```

Comment:
Correct me if I am wrong (you probably have this fix on your env, so if you can double check), but I think when adding latitude and longitude to state attributes, it gets added to the Map (in the sidebar) and afaik there is no way to hide entities. So maybe use "lat" and "long" for naming and then you can use state attribute translations to give it the full name

Reply:
Good to know... I'll fix that in the next commit.  It looks like state attribute translation might only work for `climate` entities right now (at least it doesn't work for me in my test env) so I will probably just leave as `lat` and `long` for the time being.

Reply:
It work fine for sensor attributes as well so I suggest you implement it right away.

Reply:
OK, I sorted it out, should be good now

Reply:
If you mark it ready for review I'll approve it :)